platform/linux-generic/odp_packet.c

```diff
@@ -1847,7 +1848,11 @@ static uint32_t packet_sum_crc32c(odp_packet_hdr_t *pkt_hdr,
 	return sum;
 }
 
-/** Parser helper function for Ethernet packets */
+#define PARSE_ETH_BYTES (sizeof(_odp_ethhdr_t) + 8 + 2 * sizeof(_odp_vlanhdr_t))
```

Comment:
Especially the `+8` here could use a comment, even if it's not unclear where it comes from.